src/rigid_pendulum_poc/quaternion.h

```diff
@@ -8,6 +8,8 @@
 
 namespace openturbine::rigid_pendulum {
 
+using RotationMatrix = std::tuple<Vector, Vector, Vector>;
```

Comment:
Good for now, but I think we'll want to refactor this at some point (similar to how you did the vector).